staging/src/kubevirt.io/client-go/kubecli/version.go

```diff
@@ -54,7 +55,10 @@ func (v *ServerVersion) Get() (*version.Info, error) {
 
 	var group metav1.APIGroup
 	// First, find out which version to query
-	uri := ApiGroupName
+	re := regexp.MustCompile(`(.*)/apis.*`)
+	submatchall := re.FindAllStringSubmatch(v.restClient.Get().URL().Path, -1)
+	sBasePath := submatchall[0][1]
```

Comment:
Ideally, `submatchall` needs to be null-checked. But maybe a simple `re.FindString` would be more appropriate here?

```go
sBasePath := re.FindString(v.restClient.Get().URL().Path)
```